frontend/packages/knative-plugin/src/models.ts

```diff
@@ -47,6 +47,24 @@ export const KnativeServingModel: K8sKind = {
   color: knativeServingColor.value,
 };
 
+export const KnativeServingModelV1Beta1: K8sKind = {
```

Comment:
can you mark `export const KnativeServingModel: K8sKind` as deprecated? 

and shall we rename `KnativeServingModel` as `KnativeServingModelV1Alpha1` and `KnativeServingModelV1Beta1` as `KnativeServingModel`